.github/workflows/master.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Bump version and make release job'
+on:
+  push:
+    branches:
+      - 'master'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+
+  version-bump:
+    name: 'Version Bump'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check out code'
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.JENKINS_GITHUB_PAT }}
+          # fetch-depth 0 means deep clone the repo
+          fetch-depth: 0
+      - name: 'Update Version'
+        run: |
+          set -x
+          git config user.name devops
+          git config user.email devops@runtimeverification.com
+          git checkout -B release origin/release
+          old_master="$(git merge-base origin/master origin/release)"
+          new_master="$(git rev-parse origin/master)"
+          if git diff --exit-code ${old_master} ${new_master} -- package/version; then
+            git merge --no-edit origin/master
+            ./package/version.sh bump
+          else
+            git merge --no-edit --strategy-option=theirs origin/master
+          fi
+          ./package/version.sh sub
+          if git add --update && git commit --no-edit --allow-empty --message "Set Version: $(cat package/version)"; then
+            git push origin release
+          fi
```

Comment:
I thought we would only bump versions on `release`, and leave `master` with `0.1.0` forever?
Maybe I misunderstood that...